### PR TITLE
DOC: Adds missing documentation for a kwarg.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

File: dipy/stats/analysis.py [hunk 1]
```diff
@@ -160,6 +160,10 @@ def gaussian_weights(bundle, n_points=100, return_mahalnobis=False,
     n_points : int, optional
         The number of points to resample to. *If the `bundle` is an array, this
         input is ignored*. Default: 100.
+    stat : callable, optional.
```
Comment (skoudoro) [inline dipy/stats/analysis.py:163]: Can you add the `return_mahalnobis` docstring also ?

Thank you

**Review by skoudoro — COMMENTED** [review]
Thanks @arokem,

see below a small comment.

File: dipy/stats/analysis.py (hunk 1 above)
Comment (arokem) [reply dipy/stats/analysis.py:166]: Oh yeah - good catch! Implemented in https://github.com/dipy/dipy/pull/2683/commits/a20b113e2e5faf75bf985f0c5b5d7c67c079eba7

**Comment by skoudoro** [comment]
Thanks ! merging

**Pull request merged**